### PR TITLE
Rubocop warning removal

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR works to remove a few warnings RuboCop warns about.

### CODE REVIEW

File: lib/how_is/report/html.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def header(content)
     end
 
     def link(content, url)
-      %[<a href="#{url}">#{content}</a>]
+      %(<a href="#{url}">#{content}</a>)
```
Comment (duckinator) [inline lib/how_is/report/html.rb]: as @houndci-bot pointed out, I actually had changed the configuration to use []. I'm not sure how/why RuboCop would've been giving a warning about this. O.o

File: lib/how_is/report/base_report.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@
 class HowIs
   ##
   # Subclasses of BaseReport represent complete reports.
-  class BaseReport < Struct.new(:analysis)
+  BaseReport = Struct.new(:analysis) do
```
Comment (duckinator) [inline lib/how_is/report/base_report.rb]: this can be handled separately; don't worry about it.

File: lib/how_is/pulse.rb [hunk 3]
```diff
@@ -23,7 +23,7 @@ def text_summary
     def html_summary
       parts =
         @pulse_page_response.body
-          .split('<div class="section diffstat-summary">')
+                            .split('<div class="section diffstat-summary">')
```
Comment (duckinator) [inline lib/how_is/pulse.rb]: you did this in both PRs. would you mind reverting this specific line change in this PR so it doesn't conflict with the other one?

**Comment by olleolleolle** [comment]
We're green!

**Comment by duckinator** [comment]
🎉

**Pull request merged**